### PR TITLE
feat(performance-trace-details): Adding trace level information.

### PR DESCRIPTION
**Pull request by Abdkhan14**
This pr adds trace level information under the feature flag `organizations:performance-trace-details`:
- User info + browser
- Event/Issues/Total Duration
- Web vitals (will add CTAs later)
- Trace level tag summary (we can add show tags functionality later)
- Service breakdown + replay preview

### CODE REVIEW

File: static/app/views/performance/traceDetails/newTraceDetailsTraceView.tsx [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+function generateBounds(traceInfo: TraceInfo) {
+  return boundsGenerator({
+    traceStartTimestamp: traceInfo.startTimestamp,
+    traceEndTimestamp: traceInfo.endTimestamp,
+    viewStart: 0,
+    viewEnd: 1,
+  });
+}
+
+export default function NewTraceView({
```
Comment (Abdkhan14) [inline static/app/views/performance/traceDetails/newTraceDetailsTraceView.tsx:138]: Added a new file for the traceview,  to isolate changes from having effect on the current traceview. No major changes right now just just prop changes and renders the new TraceViewHeader.

File: static/app/views/performance/traceDetails/newTraceDetailsContent.tsx [hunk 2]
```diff
@@ -121,54 +154,123 @@ function NewTraceDetailsContent(props: Props) {
...
+
+    if (roots > 1) {
+      return TraceType.MULTIPLE_ROOTS;
+    }
+
+    if (orphanErrors && orphanErrors.length > 1) {
+      return TraceType.ONLY_ERRORS;
+    }
+
+    return TraceType.ONE_ROOT;
```
Comment (wmak) [inline static/app/views/performance/traceDetails/newTraceDetailsContent.tsx:173]: nit: should we do `elseif roots === 1` here and throw an error if we get to the else statement? cause for example you could have `roots === 0 and orphans === 0` as a potential case that's currently not captured

**Review by wmak — APPROVED** [review]

File: static/app/components/events/interfaces/spans/newTraceDetailsHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    return null;
+  }
+
+  const projectSlug = event.projectSlug;
+  const project = projects.find(proj => proj.slug === projectSlug);
+
+  const isFrontendRoot =
+    project &&
+    platformToPerformanceType(projects, [Number(project?.id)]) ===
+      ProjectPerformanceType.FRONTEND &&
```
[the remark this answers is not in the document]
Comment (Abdkhan14) [reply static/app/components/events/interfaces/spans/newTraceDetailsHeader.tsx:99]: Yep, makes sense to me.

[the remark this answers is not in the document]
Comment (Abdkhan14) [reply static/app/components/events/interfaces/spans/newTraceDetailsHeader.tsx]: Yep makes sense to me.

**Pull request merged**